src/sentry/runner/commands/devservices.py

```diff
@@ -63,6 +63,10 @@ def up(project, exclude):
     # services are run if they're not needed.
     if not exclude:
         exclude = set()
+
+    if not ('BIGTABLE_EMULATOR_HOST' in os.environ and 'bigtable' in settings.SENTRY_NODESTORE):
```

Comment:
Using the presence of the environment variable rather than just checking the backend (and letting it break if the host is not set?) seems weird and not consistent with the other backends.

Reply:
So, this specifically relates to the emulator. Just having the backend declared won't be enough to get the internal `bigtable` client to work, unfortunately. And there's no configuration for the client to say "hey, use the emulator" except this environment variable that's set. :(

Reply:
Did it.